server.js

```diff
@@ -78,9 +78,9 @@ app.prepare().then(async () => {
   server.get("/_", async (req, res) => {
     const isBucketsAvailable = await Utilities.checkTextile();
 
-    if (!isBucketsAvailable) {
-      return res.redirect("/maintenance");
-    }
+    // if (!isBucketsAvailable) {
```

Comment:
Don't forget on a release PR to make sure that this code is put back. its important!